packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3489,6 +3504,25 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     return chain.join(' \u2190 ');
   }
 
+  /// Returns the first user created widget ancestor.
+  ///
+  /// Only available in debug builds with flag `--track-widget-creation`.
+  ///
+  /// Useful for debugging the source of an element.
```

Comment:
make this method private or consider merging with the describeFirstLocalWidget one line above.
In general we should try to keep API surfaces on main classes as compact as feasible.